### PR TITLE
Some more cleanup

### PR DESCRIPTION
**Pull request by XWB**

### CODE REVIEW

File: src/Manager/RepositoryManagerInterface.php [hunk 1]
```diff
@@ -26,8 +26,7 @@ interface RepositoryManagerInterface
      * Adds index name and its finder.
      * Custom repository class name can also be added.
      *
-     * @param string $indexName      The type name in "index/type" format
-     * @param string $repositoryName
+     * @param string $indexName The type name in "index/type" format
```
Comment (deguif) [inline src/Manager/RepositoryManagerInterface.php:29]:
```suggestion
     * @param string $indexName The type name in "index" format
```

But useless I think

**Review by deguif — APPROVED** [review]

File: src/Manager/RepositoryManagerInterface.php [hunk 2]
```diff
@@ -25,9 +25,6 @@ interface RepositoryManagerInterface
     /**
      * Adds index name and its finder.
      * Custom repository class name can also be added.
-     *
-     * @param string $indexName      The type name in "index/type" format
-     * @param string $repositoryName
      */
     public function addIndex(string $indexName, FinderInterface $finder, string $repositoryName = null): void;
```
Comment (deguif) [inline src/Manager/RepositoryManagerInterface.php:29]:
```suggestion
    public function addIndex(string $indexName, FinderInterface $finder, ?string $repositoryName = null): void;
```

Comment (deguif) [reply src/Manager/RepositoryManagerInterface.php:29]: There is a CS rule for that: `nullable_type_declaration_for_default_null_value`

Comment (deguif) [reply src/Manager/RepositoryManagerInterface.php:29]: I remember adding it to ruflin/elastica ;)

Comment (XWB) [reply src/Manager/RepositoryManagerInterface.php]: Nice. PR has been updated :)

**Pull request merged**